src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -121,4 +121,42 @@ public boolean isUserRegistered(String username) {
   public void setUsers(List<User> users) {
     this.users = users;
   }
+
+  /** Returns the number of users in UserStore */
+  public int numUsers() {
+    if (users != null) {
+      return users.size();
+    }
+    return 0;
+  }
+
+  /** Returns the last user added */
+  public User getLastUser() {
+    if (!users.isEmpty()) {
+      return users.get(users.size() - 1);
+    }
+    return null;
+  }
+
+  /** Returns the user with flipped is_admin flag */
+  public User setIsAdmin(User user, boolean is_admin) {
```

Comment:
Instead of creating a new User instance to write with, we should be directly reading the User and modifying that specific instance. Is it possible to use the uuid as a key on writes to make this easier to read and write?

Reply:
I modified the code to pass in user itself and is_admin as parameter instead. Where do you use uuid as the key on writes? I currently have Key key = KeyFactory.createKey("chat-users", uuid) which uses uuid as the key.